ledger/participant-integration-api/src/test/suite/scala/platform/store/ConversionsSpec.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store
+
+import com.daml.ledger.api.domain
+import com.daml.ledger.participant.state.{v1 => state}
+import com.daml.platform.store.Conversions._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
+
+class ConversionsSpec extends AsyncWordSpec with Matchers {
```

Comment:
@fabiotudone-da: Tests are here.

Reply:
How about the `toStateV1RejectionReason` conversions?